helix-core/src/main/java/org/apache/helix/task/WorkflowConfig.java

```diff
@@ -94,6 +95,16 @@ public WorkflowConfig(WorkflowConfig cfg, String workflowId) {
         cfg.isAllowOverlapJobAssignment(), cfg.getTimeout());
   }
 
+  @VisibleForTesting
```

Comment:
Let us try use reflection to change it. And later remove the change once fix is in.